docs/users.rst

```diff
@@ -53,29 +53,27 @@ text file (this means you can view the information by using *cat* command.)
     tss:x:59:59:Account used by the trousers package to sandbox the tcsd daemon:/dev/null:/sbin/nologin
 
 
-Each line has seven entries separated by *:*. 
-
-::
+Each line has seven entries separated by::
```

Comment:
This is missing the vital **:** part, which is the separator.

Reply:
For Sphinx below two things are same:
```
Each line has seven entries separated by::
```
```
Each line has seven entries separated by:

::
```

When we generate output, Sphinx will automatically expands ending `::`, with `:` followed by a `::` code-directive, see below screenshot. In output, we're still getting a `:` at the end of sentence.

![users](https://user-images.githubusercontent.com/28824347/126466124-f09fc308-b2e9-4422-8965-8b32708f5c7d.png)